include/tvm/tir/transform.h

```diff
@@ -458,6 +458,14 @@ TVM_DLL Pass FlattenBuffer();
  */
 TVM_DLL Pass TextureFlatten();
 
+/*!
+ * \brief Implements a Common Subexpression Elimination (CSE)
+ *        which introduces let-in bindings for duplicated sub-expressions.
+ * \param enable_cse_tir Whether common subexpression elimination is enabled.
+ * \return The pass.
+ */
+TVM_DLL Pass CommonSubexprElim(bool enable_cse_tir = true);
```

Comment:
Please choose another name to make it explicit that this pass is for TIR, to avoid confusion with the existing relay-level CSE `EliminateCommonSubexpr`.

Reply:
At first I was about to make it explicit in the name that it's for TIR, and in the end decided otherwise as the file is in the /tir/ subfolder.
But you're right, it's probably better to make that explicit in the name!

Reply:
I see, since it is under `tvm::tir::transform` namespace, we might not need to say TIR in the function name. Up to you.

Reply:
I'll change it with a "TIR" suffix, to make that clear :)